tests/end-to-end/director/runtime_api_test.go

```diff
@@ -357,9 +357,16 @@ func TestQueryRuntimes(t *testing.T) {
 		}
 	}()
 
+	inputRuntimes := []*graphql.Runtime{
+		{Name: "runtime-query-1", Description: ptrString("test description")},
+		{Name: "runtime-query-2", Description: ptrString("another description")},
+		{Name: "runtime-query-3"},
+	}
+
 	for i := 0; i < 3; i++ {
```

Comment:
iterate over inputRuntimes, if we change it to only 2 elements we will get panic